src/unix/apple/cpu.rs

```diff
@@ -371,6 +382,29 @@ pub(crate) fn get_vendor_id_and_brand() -> (String, String) {
     (vendor, get_sysctl_str(b"machdep.cpu.brand_string\0"))
 }
 
+pub(crate) unsafe fn get_cpu_arch() -> CpuArch {
+    use std::ffi::CStr;
+    let mut mib: [c_int; 2] = [libc::CTL_HW, libc::HW_MACHINE_ARCH];
+    let mut arch_str: [u8; 32] = [0; 32];
+
+    if get_sys_value(
+        libc::CTL_HW as _,
+        libc::HW_MACHINE as _,
+        mem::size_of::<[char; 32]>(),
+        arch_str.as_mut_ptr() as *mut _,
+        &mut mib,
+    ) {
+        CStr::from_bytes_until_nul(&arch_str)
```

Comment:
Please update the minimum rust version to 1.69 so we can use this method.

Reply:
Is change in Cargo.tom enough or is there some CI configuration to change on GitHub ?